src/odemis/driver/xt_client.py

```diff
@@ -1247,36 +1205,34 @@ def start_autostigmating_flash(self):
         """
         with self._proxy_access:
             self.server._pyroClaimOwnership()
-            return self.server.start_autostigmating_flash()
+            self.server.start_autostigmating_flash()
 
-    def start_autostig(self, n_images=17, sweep_range=0.015, dwell_time=3e-6, resolution=(1536, 1024),
-                       reduced_area_size=0.5, save_results=False, results_fp=None):
+    def start_autostig(self, n_images: int = 17, sweep_range: float = 0.015, dwell_time: float = 3e-6,
+                       resolution: Tuple[int, int] = (1536, 1024),
+                       reduced_area_size: float = 0.5, save_results: bool = False,
+                       results_fp: Optional[Union[Path, str]] = None) -> Tuple[float, float]:
```

Comment:
The return seems to be just float.

Reply:
It is a tuple of x and y, I've updated the docstring to clarify